src/content/docs/distributed-tracing/infinite-tracing/set-trace-observer.mdx

```diff
@@ -153,16 +295,70 @@ To set up a trace observer:
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="multi-named-apps"
+    title="What if my browser app is connected to an APM agent that has multiple names?"
+  >
+    If your browser app is associated with an [APM application that has multiple names](/docs/apm/agents/manage-apm-agents/app-naming/use-multiple-names-app/), you can only select one from the group of multi-named browser apps when you choose your data source. If you try to select the same application that goes by another name, the UI will prompt you that you need to choose only one browser app.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="lambda-sampling"
+    title="How does Lambda sampling work?"
+  >
+  By default, when you set up Lambda layers, you automatically get head-based sampling. When you enable Infinite Tracing for Lambda, your head-based sampling data is additionally sampled by the trace observer. The main benefit to enabling Infinite Tracing for your Lambda functions is to ensure their spans are included with spans from other services tracked by Infinite Tracing. 
+  </Collapser>
+</CollapserGroup>
+ 
+The instructions below are about using the New Relic UI, but if you prefer to take a programmatic approach, you can specify your trace observer data sources using [GraphQL](/docs/apis/nerdgraph/examples/configure-infinite-tracing-graphql/#updating-the-data-sources). When you finish, return to this page and continue with step 4 (Optional configurations) below. 
+
+To finish the configuration for browser, mobile, and Lambda in the UI: 
+
+1. Return to the trace observer Nerdlet (**[one.newrelic.com](https://one.newrelic.com) > Apps > Your Apps > New Relic Edge**).
+
+2. For your trace observer, go to the end of the row, click the three dots, and then click **Edit mobile, browser, lambda sources**.
+
+  ![Screenshot showing menu when you click the three dots next to your trace observer](./images/browser-mobile-lambda-menu.png "Screenshot showing menu when you click the three dots next to your trace observer")
+
+3. Select the data sources you want to include, keeping the following in mind:
+
+    * You can use the filter to narrow your options.
+    * If your browser app is connected to an APM agent has multiple names, you can can only choose one name.
+    * If an entry is gray, it is already being used by another trace observer.
+
+  ![Screenshot showing the list of data sources you want to send tracing data](./images/select-data-sources.png "Screenshot showing the list of data sources you want to send tracing data")
+  
+4. Bookmark the [optional configurations](#follow-up-config) below so you can return to them after you experiment with Infinite Tracing.
+
+5. View your Infinite Tracing data in the [New Relic UI](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui/).
```

Comment:
a small thing, but I notice some of your link text isn't that descriptive. for this example, I would go something like: 

View your data in the [distributed tracing UI], 
instead of '[New Relic UI]. 

I think this is keeping with our general guidelines to try to make the link text slightly descriptive of what's there. In other words, [New Relic UI] I might suppose would take me to general info about New Relic One UI or something. I only mention cause I saw a couple other link texts that were pretty vague like that.

Reply:
@zuluecho9, good tip! I fixed this one and some other generally worded links!